### PR TITLE
Add c2patool

### PR DESCRIPTION
**Pull request by leszko**

### CODE REVIEW

**Comment by leszko** [comment]
> Note that this will get it in the Docker image but not in the tarball archive that will run during a deploy - that'll need to happen via something like [2f584db](https://github.com/livepeer/catalyst/commit/2f584db98524b577bd8a8449f836e0417c3ff06f).
> 
> That said, this is making me feel like we should probably just generate that archive off of the Docker image anyway for simplicity

Thanks for the comment. Right, generating the archive from Docker image would be great. 

Though, for this PR, I'll just add what you pointed out.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -77,6 +77,14 @@ jobs:
 
           cp ./scripts/livepeer-vmagent ./bin/livepeer-vmagent
 
+      - name: Build c2patool
+        run: |
+          apt update && apt install -y curl build-essential git
+          curl https://sh.rustup.rs -sSf | sh -s -- -y
+          source "$HOME/.cargo/env"
+          cargo install c2patool
```
Comment (iameli-streams) [inline .github/workflows/build.yaml:85]: Probably this to match the other one?
```suggestion
          cargo install --version 0.6.2 c2patool
```

**Review by iameli-streams — APPROVED** [review]
Approved with tiny change

File: .github/workflows/build.yaml (hunk 1 above)
Comment (pwilczynskiclearcode) [reply .github/workflows/build.yaml:85]: https://github.com/marketplace/actions/cargo-install I find dedicated github actions working faster than manual installations. Maybe worth trying out

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -77,6 +77,16 @@ jobs:
 
           cp ./scripts/livepeer-vmagent ./bin/livepeer-vmagent
```
Comment (pwilczynskiclearcode) [inline .github/workflows/build.yaml:79]: https://github.com/actions-rs/toolchain
```
      - uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
```
might be needed before using `cargo`

Comment (leszko) [reply .github/workflows/build.yaml:79]: Thanks, trying it out!

**Review by pwilczynskiclearcode — APPROVED** [review]

**Comment by leszko** [comment]
Thanks @pwilczynskiclearcode for the suggestion with the cargo GH Action!

**Pull request merged**